app/views/users/show.html.erb

```diff
@@ -1 +1,26 @@
-<h1 name='title'>Here is most of the information for User ID: <%= @user_id %></h1>
\ No newline at end of file
+<head>
+  <%= stylesheet_link_tag "user_show" %>
+</head>
+<body class="mrg">
+  <div>
+    <%= render partial: 'user', locals: { user: @user } %>
+  </div>
+  <div class="border mrg">
+    <div class="title mrg">
+      <h2>Bio</h2>
+      <p><%= @user.bio %></p>
+    </div>
+  </div>
+  <div>
+    <% if @posts.present? %>
+      <% @posts.each_with_index do |post, index| %>
```

Comment:
- [ ] Please use the `most_recent_post` method you created in your User model to render the 3 most recent posts as mentioned in the project requirements.